odoo/tools/mail.py

```diff
@@ -189,16 +189,13 @@ def html_sanitize(src, silent=True, sanitize_tags=True, sanitize_attributes=Fals
     }
     if sanitize_tags:
         kwargs['allow_tags'] = allowed_tags
-        if etree.LXML_VERSION >= (2, 3, 1):
-            # kill_tags attribute has been added in version 2.3.1
-            kwargs.update({
-                'kill_tags': tags_to_kill,
-                'remove_tags': tags_to_remove,
-            })
-        else:
-            kwargs['remove_tags'] = tags_to_kill + tags_to_remove
+        # kill_tags attribute has been added in version 2.3.1
+        kwargs.update({
+            'kill_tags': tags_to_kill,
+            'remove_tags': tags_to_remove,
+        })
```

Comment:
```suggestion
        kwargs['kill_tags'] = tags_to_kill
        kwargs['remove_tags'] = tags_to_remove
```
while at it? Or at the very least remove the useless inner dict literal and use kwargs?